gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationViewModel.java

```diff
@@ -249,33 +249,18 @@ private Completable handlePhotoCaptureResult(String fieldId) {
   private Completable saveBitmapAndUpdateResponse(Bitmap bitmap, String fieldId)
       throws IOException {
     File file = fileUtil.saveBitmap(bitmap, fieldId + ".jpg");
-    String destinationPath = getRemoteImagePath(file.getName());
-
-    // If offline, Firebase will automatically upload the image when the network
-    // connectivity is  re-established.
-    // TODO: Implement offline photo sync using Android Workers and local db
-    String url = firestoreStorageManager.uploadMediaFromFile(file, destinationPath);
+    String destinationPath =
```

Comment:
It would seem saving the bitmap to local storage is out of scope of the viewmodel's responsibilities. Ideally both this and the access to the worker would be managed by the storageManager. Could these be moved there, or a TODO added to do so?